src/utils/apiCalls.js

```diff
@@ -1,65 +1,64 @@
 const { version } = require('../../config.json');
```

Comment:
`getBaseUrl()` is not supposed to exist anymore. I vividly remember fixing this in one of my PRs, but a merge must have broken it. The line is supposed to be
```js
const baseUrl = `https://discord.com/api/${version}`
```